extensions/wikia/AchievementsII/services/AchAwardingService.class.php

```diff
@@ -612,38 +613,38 @@ private function awardNotInTrackBadge( $badge_type_id, $badge_lap = null ) {
 
 		$notInTrackStatic = AchConfig::getInstance()->getNotInTrackStatic();
 
-		if ( isset( $notInTrackStatic[$badge_type_id] ) ) {
-			$badge_level = $notInTrackStatic[$badge_type_id]['level'];
+		if( isset( $notInTrackStatic[ $badge_type_id ] ) ) {
+			$badge_level = $notInTrackStatic[ $badge_type_id ][ 'level' ];
 		} else {
 			$badge_level = BADGE_LEVEL_PLATINUM;
 		}
 
 		$this->mNewBadges[] = array( 'badge_type_id' => $badge_type_id,
-									'badge_lap' => $badge_lap,
-									'badge_level' => $badge_level );
+			'badge_lap' => $badge_lap,
+			'badge_level' => $badge_level );
 
-		if ( !isset( $this->mBadges[$badge_type_id] ) ) {
-			$this->mBadges[$badge_type_id] = 0;
+		if( !isset( $this->mBadges[ $badge_type_id ] ) ) {
+			$this->mBadges[ $badge_type_id ] = 0;
 		}
-		$this->mBadges[$badge_type_id]++;
+		$this->mBadges[ $badge_type_id ]++;
 
-		if ( $badge_type_id == BADGE_WELCOME ) {
-			if ( !isMsgEmpty( 'welcome-user-page' ) ) {
+		if( $badge_type_id == BADGE_WELCOME ) {
+			if( !isMsgEmpty( 'welcome-user-page' ) ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Merge this if statement with the enclosing one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1066)